### DISCUSSION TITLE
Offcanvas component in Ng-clarity

### DISCUSSION THREAD

**Discussion by kumar-tadepalli** [vmware-clarity/ng-clarity#532] [Q&A]
Hi,

Do we have a Offcanvas component in Ng-clarity?
Examples:
https://ng-bootstrap.github.io/#/components/offcanvas/examples

Thanks
Kumar

**Comment by Jinnie** [comment]
No. It looks like a modal with a more special positioning.

CC: attn. PM @pdaigle
